### DISCUSSION TITLE
Shared graphcache with multiple Microfrontends and clients architecture

### DISCUSSION THREAD

**Discussion by dennis-hammarstrand-stratsys** [urql-graphql/urql#2362] [Q&A]
Hello! 

Our use case is an application that has it's own instance of an urql client and then reads in multiple Microfrontends that each has their own instance of an urql client, desireably we would want to share the same graphcache instance to take advantage of the "global" state benefits that it comes with.
Is this at all possible or do you need to have a central urql client instance which you then need to distribute somehow to the underlying Microfrontends? Or could you since they are all wrapped in the same "main" application take advantage of the graphcache instance somehow?

**Comment by gferreri** [comment]
This is becoming a critical missing feature for my org as well. I'd be interested in contributing to help make this a reality!
